src/main/scala/rocket/HellaCache.scala

```diff
@@ -51,8 +40,8 @@ trait HasL1HellaCacheParameters extends HasCacheParameters with HasCoreParameter
 
   require(isPow2(nSets))
   require(rowBits >= coreDataBits)
-  require(rowBits <= outerDataBits)
-  require(xLen <= outerDataBits) // would need offset addr for puts if data width < xlen
+  require(rowBits == cacheDataBits) // TODO should rowBits even be seperably specifiable?
```

Comment:
I think we want to keep this option, since the RAM dimensions can serve as an energy/performance tuning parameter.